.github/workflows/release.yml

```
@@ -533,13 +533,15 @@ jobs:
           latest=false
           suffix=-${{ matrix.label }}
 
+    - name: Install regctl
+      uses: regclient/actions/regctl-installer@main
```

```suggestion
      uses: regclient/actions/regctl-installer@b6614f5f56245066b533343a85f4109bdc38c8cc
```

We should pin 3rd party actions dependency as suggested by GitHub's security guidelines.